chimney/src/test/scala/io/scalaland/chimney/WixSpec.scala

```diff
@@ -0,0 +1,206 @@
+
+        (colors4.BLUE: colors4.Color).transformInto[colors2.Color] ==> colors2.Blue
+        (colors4.GREEN: colors4.Color).transformInto[colors2.Color] ==> colors2.Green
+      }
+
+      "transforming enum ignoring underscore" - {
+        (colorsUpperUndescore.BLOODY_RED: colorsUpperUndescore.Color)
+          .transformInto[colorsUpperCamel.Color] ==> colorsUpperCamel.BloodyRed
+        (colorsUpperUndescore.DARK_GREEN: colorsUpperUndescore.Color)
+          .transformInto[colorsUpperCamel.Color] ==> colorsUpperCamel.DarkGreen
+
+        (colorsUpperCamel.SkyBlue: colorsUpperCamel.Color)
+          .transformInto[colorsUpperUndescore.Color] ==> colorsUpperUndescore.SKY_BLUE
+        (colorsUpperCamel.SnowWhite: colorsUpperCamel.Color)
+          .transformInto[colorsUpperUndescore.Color] ==> colorsUpperUndescore.SNOW_WHITE
+      }
+    }
+
+    "support java enum" - {
+      "transform java enum into java enum" - {
+        "by canonical name" - {
+          implicit def t[A]: Transformer[JavaNumbers.NumScaleUppercase, JavaNumbers.NumScale] =
+            Transformer.define.buildTransformer
+
+          (JavaNumbers.NumScaleUppercase.ZERO: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Zero
+          (JavaNumbers.NumScaleUppercase.MILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Million
+          (JavaNumbers.NumScaleUppercase.BILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Billion
+          (JavaNumbers.NumScaleUppercase.TRILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Trillion
+        }
+        "with customization" - {
+          implicit def t[A]: Transformer[JavaNumbers.NumScaleUppercase, JavaNumbers.NumScale] =
+            Transformer.define
+              .withCoproductInstance(JavaNumbers.NumScaleUppercase.TRILLION, JavaNumbers.NumScale.Zero)
+              .buildTransformer
+
+          (JavaNumbers.NumScaleUppercase.ZERO: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Zero
+          (JavaNumbers.NumScaleUppercase.MILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Million
+          (JavaNumbers.NumScaleUppercase.BILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Billion
+          (JavaNumbers.NumScaleUppercase.TRILLION: JavaNumbers.NumScaleUppercase)
+            .transformInto[JavaNumbers.NumScale] ==> JavaNumbers.NumScale.Zero
+        }
+      }
+
+      "transform java enum into sealed hierachy" - {
+        "by exact name" - {
+          implicit val t: Transformer[Colors, colors2.Color] = Transformer.define.buildTransformer
```

Comment:
is this line redundant?

Reply:
The transformer can be derived for each transformation independently. I prefer when we declare it like this though, makes it clear, that this specific `Transformer` is used. If we omit it, then while reading the code, there's always a possibility that transformer may be declare elsewhere rather than derived automatically.

If you insist, I can remove this.